ext/standard/var_unserializer.re

```diff
@@ -638,6 +638,18 @@ declared_property:
 					}
 				}
 			} else {
+				if (UNEXPECTED(obj->ce->ce_flags & ZEND_ACC_NO_DYNAMIC_PROPERTIES)) {
```

Comment:
This block should be moved below `int ret = is_property_visibility_changed(obj->ce, &key);` and the if/elseif checking if the property visibility changed from public to private/protected